lib/babe/babe.go

```diff
@@ -497,6 +497,21 @@ func (b *Service) handleSlot(epoch, slotNum uint64,
 		return err
 	}
 
+	slotNumOfTheBestBlock, err := b.blockState.GetSlotForBlock(parentHeader.Hash())
```

Comment:
nit rename to `bestBlockSlotNum`? `slotNumOfTheBestBlock` feels a bit poetically verbose ✍️

Reply:
addressed